util/flags.py

```diff
@@ -24,6 +24,16 @@ def create_flags():
     # Data Augmentation
     # ================
 
+    f.DEFINE_string('train_augmentation_files', '', 'comma separated list of files or dirs, specifying the dataset used for mixing audio into train dataset, if empty, no mix will be run in train phase')
+    f.DEFINE_string('dev_augmentation_files', '', 'comma separated list of files or dirs, specifying the dataset used for mixing audio into dev dataset, if empty, no mix will be run in dev phase')
+    f.DEFINE_string('test_augmentation_files', '', 'comma separated list of files or dirs, specifying the dataset used for mixing audio into test dataset, if empty, no mix will be run in test phase')
+    f.DEFINE_float('audio_aug_min_audio_dbfs', 0, 'min value of dbfs to specify the min volume of audio during gaining audio')
```

Comment:
> I tired some different parameters, mixing my speech dataset as noise.
> The parameters which i found best (I listened to some augmented example audiofiles) were:
> 
> ```
> --audio_aug_mix_noise_min_audio_dbfs -5 
> --audio_aug_mix_noise_max_audio_dbfs -30 
> --audio_aug_mix_noise_min_snr_db 10 
> --audio_aug_mix_noise_max_snr_db 30
> ```
> 
> I also tried a combination with `augmentation_pitch_and_tempo_scaling` which makes the audio hard to understand and `augmentation_freq_and_time_masking` with which i had to listen twice most times to understand the text.

* Because `augmentation_xxx` are spectrogram augmentations, to reconstruct them back to audio could cause heavy distortion, and I'm not familiar which this domain, so I just implement the basic Griffim-Lim Argorithm, I don't know how to improve it at present.

* Oh, I wrote them in exchange, I will fix it and note it, fortunately `tf.random_uniform`'s `minval` and `maxval` does not care about misplaced value.